living_documentation_regime/action_inputs.py

```diff
@@ -67,7 +68,17 @@ def get_liv_doc_regime() -> bool:
         Getter of the LivDoc regime switch.
         @return: True if LivDoc regime is enabled, False otherwise.
         """
-        return get_action_input(LIV_DOC_REGIME, "false").lower() == "true"
+        return get_action_input(Regime.LIV_DOC_REGIME, "false").lower() == "true"
+
+    @staticmethod
+    def get_output_formats() -> list[str]:
+        """
+        Getter of the output formats for generated documents.
+        @return: A list of output formats.
+        """
+        output_formats_json = get_action_input(OUTPUT_FORMATS, "[]").strip().lower()
```

Comment:
Missing usage in Full example in README.md.
Why json format? Why not to use simple list of string values?
Example: 'mdoc, pdf'


You already said in action.yml:
- description: 'Comma-separated list of wanted output formats.'

Reply:
Your example is a string, please can you write down full example? Because in my opinion, there can not be written input as a `export INPUT_OUTPUT_FORMATS=[mdoc, pdf]` or `export INPUT_OUTPUT_FORMATS=["mdoc", "pdf"]`, because it returns the type of string. JSON is in my opinion the best way how to effectively handle the list of string as input in yaml file. Do you agree and should I change the action.yml description of an input?

Reply:
After sync, the logic for handling output formats was improved. See commit: [7f72ae4](https://github.com/AbsaOSS/living-doc-generator/pull/83/commits/7f72ae44b2ca610922c4d812b551b81195bb5648).